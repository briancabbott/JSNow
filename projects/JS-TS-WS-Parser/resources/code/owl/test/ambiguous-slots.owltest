---a = b c b
b = 'x' | ''
c = ''
