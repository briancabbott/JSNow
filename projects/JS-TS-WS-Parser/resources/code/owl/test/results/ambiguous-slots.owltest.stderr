error: this grammar is ambiguous

. x 

  can be parsed in two different ways: as

. x     
  b c b 
  a---- 

  or as

.     x 
  b c b 
  a---- 

